static/js/index.js

```diff
@@ -155,3 +158,10 @@ window.onclick = function (event) {
     parpopup.style.display = 'none';
   }
 };
+// ========================================================================
```

Comment:
[OPTIONAL] It is not a good practice to use this commented lines  to diferentiate or segment your code since it becomes bulky and occupies unnecessary space as your project become bigger. :pray:  You can consider breaking the code into smaller components and using expressive and meaningful naming conventions for variables and functions instead which is a better practice and easier to read and understand. :+1:

Reply:
- Thanks 🙏 for your review I appreciate your effort to review the pull request 😊

- Have a nice day 😊